dashboard/README.md

```diff
@@ -0,0 +1,32 @@
+SeaClouds Dashboard
+==================
+
+This component provides an easy way to interact with SeaClouds Platform by using an Angular.js web application. 
+
+# Installing and running SeaClouds Dashboard
+The easiest way to install and run this component is explained in SeaClouds [README.md](../README.md). However, there are some
+use case that may find interesting to run SeaClouds Dashboard in standalone mode, like adding new features or running SeaClouds Dashboard
+wired with a remote SeaClouds Platform instance.
+
+## Requirements
+Running SeaClouds Dashboard only requires Java 7 (or greater) installed on the target machine. Among with the software requirements, it also needs:
+
+- SeaClouds Planner, Sla, Monitor and Deployer Components up and running on an accessible endpoint.
+- 1 open port if you want to access to the Dashboard remotely.
```

Comment:
I think this is not clear. what do you mean by remotely? I guess one wants always to access the Dashboard